frontend_build/src/webpack.config.base.js

```diff
@@ -41,7 +41,11 @@ var config = {
       {
         test: /\.js$/,
         loader: 'babel',
-        exclude: /node_modules/
+        exclude: /node_modules/,
+        query: {
+          presets: ['es2015-ie'],
```

Comment:
Use this special preset that forces classes to be transformed in a way that makes them IE9 compatible. Without this, subclasses of `KolibriModule` were not running the parent constructor.